helmfile/charts/hnc/values.yaml

```diff
@@ -0,0 +1,112 @@
+fullnameOverride: ""
+nameOverride: ""
+
+imagePullSecrets: []
+image:
+  repository: gcr.io/k8s-staging-multitenancy/hnc-manager
+  pullPolicy: IfNotPresent
+  tag: ""
+
+serviceAccount:
+  create: true
+  annotations: {}
+  name: ""
+
+rbac:
+  create: true
+
+  user:
+    allowConfigEdit: false
+
+    namespaces:
+    - default
+
+    groups: []
+    users:
+    - admin@example.com
+
+podAnnotations: {}
+
+podSecurityContext:
+  runAsNonRoot: true
+  runAsUser: 1000
+  fsGroup: 2000
+
+securityContext:
+  allowPrivilegeEscalation: false
+  capabilities:
+    drop:
+    - ALL
+  readOnlyRootFilesystem: true
+  runAsNonRoot: true
+  seccompProfile:
+    type: RuntimeDefault
+
+includedNamespacesRegex: ""
+
+excludedNamespaces:
+- hnc-system
+- kube-node-lease
+- kube-public
+- kube-system
+
+unpropagatedAnnotations: []
+
+managedNamespaceAnnotations: []
+managedNamespaceLabels: []
+
+managerDeployment:
+  create: true
+  replicaCount: 1
+
+  resources:
+    requests:
+      cpu: 100m
+      memory: 150Mi
+    limits:
+      cpu: 100m
+      memory: 300Mi
+
+  nodeSelector: {}
+
+  tolerations: []
+
+  affinity: {}
+
+managerService:
+  create: true
+  annotations: {}
+  port: 8080
+
+webhook:
+  create: true
+  annotations: {}
+
+webhookDeployment:
+  create: false
+  replicaCount: 3
+
+  resources:
+    requests:
+      cpu: 100m
+      memory: 150Mi
+    limits:
+      cpu: 100m
+      memory: 300Mi
+
+  nodeSelector: {}
+
+  tolerations: []
+
+  affinity: {}
+
+webhookService:
+  create: true
+  port: 443
+
+certificate:
+  create: true
+
+serviceMonitor:
+  create: true
+  relabelings: []
```

Comment:
More config is often good. But would you ever want to not deploy manager-*, webhook, and webhookservice (plus maybe some others)? I'm thinking that the `create` options could be removed from some.
But maybe I'm wrong, so please let me know your thoughts on this.

Reply:
It's mostly a leftover from when I created the chart and tied it to the build process in the HNC repo. I often compared the result to their generated manifests so it was easier to do so when I could select which one to template.

Reply:
I've simplified the chart a bit now and added the numbers to the dashboard.